src/app/child-dev-project/children/children-count-dashboard/children-count-dashboard.component.ts

```diff
@@ -1,54 +1,84 @@
-import { Component, OnInit } from "@angular/core";
+import { Component, Input, OnInit } from "@angular/core";
 import { ChildrenService } from "../children.service";
 import { Router } from "@angular/router";
-import { UntilDestroy, untilDestroyed } from "@ngneat/until-destroy";
 import { OnInitDynamicComponent } from "../../../core/view/dynamic-components/on-init-dynamic-component.interface";
+import { take } from "rxjs/operators";
+import { ConfigurableEnumValue } from "../../../core/configurable-enum/configurable-enum.interface";
 
-@UntilDestroy()
 @Component({
   selector: "app-children-count-dashboard",
   templateUrl: "./children-count-dashboard.component.html",
   styleUrls: ["./children-count-dashboard.component.scss"],
 })
 export class ChildrenCountDashboardComponent
   implements OnInitDynamicComponent, OnInit {
+  /**
+   * The property of the Child entities to group counts by.
+   *
+   * Default is "center".
+   */
+  @Input() groupBy: string = "center";
+
   totalChildren: number;
-  childrenByCenter = [];
+  childrenGroupCounts = [];
```

Comment:
The typing of this array could be more explicit. As far as I got it this is a two dimensional array where the second dimension always contains `[string, number]`. So why not define an array of `{label: string, value: number}` objects?

Reply:
:+1: